<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraView</class>
 <widget class="QWidget" name="CameraView">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>559</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>660</width>
    <height>650</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="recognition_result">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>480</y>
     <width>317</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>317</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>--</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>520</y>
     <width>317</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>317</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>Bandwidth:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_image">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>640</width>
     <height>480</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>640</width>
     <height>480</height>
    </size>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_bandwidth">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>520</y>
     <width>317</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>317</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>0 kbps</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>500</y>
     <width>317</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>317</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>Framerate:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>540</y>
     <width>317</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>317</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>Residual energy:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_energy">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>540</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>100 %</string>
   </property>
  </widget>
  <widget class="QLabel" name="recognition_label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>480</y>
     <width>317</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>317</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>Object:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_frameRate">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>500</y>
     <width>317</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>317</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>0 fps</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
